content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -208,7 +209,30 @@ http://localhost:9090/graph
 
 For more details, see [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics/#about-the-prometheus-add-on).
 
-### ServiceGraph
+### Kiali and ServiceGraph
+
+Set up a tunnel to Kiali:
```

Comment:
Please use numbered lists for steps.

Reply:
for which steps?

Reply:
All steps.

Reply:
done;  that change is pretty simple but this pr isn't meant to address prexisting formatting stuff; if there are more complicated formatting stuff thats needed, please open up a separate an address them there once this is merged